doc/tutorial.md

```diff
@@ -691,6 +691,64 @@ fn calculator5() {
 }
 ```
 
+<a id="calculator7"></a>
```

Comment:
Meant to ask what it was for, I can swap calculator6 and 7 as they are now or should I remove 6?